setup.cfg

```diff
@@ -1,12 +1,12 @@
 [metadata]
 name = SPARQLWrapper
-version = attr: SPARQLWrapper.__init__.__version__
+version = attr: SPARQLWrapper.__version__
 description = SPARQL Endpoint interface to Python
 long_description = This is a wrapper around a SPARQL service. It helps in creating the query URI and, possibly, convert the result into a more manageable format.
 license = W3C SOFTWARE NOTICE AND LICENSE
-author = attr: SPARQLWrapper.__init__.__authors__
-author_email = attr: SPARQLWrapper.__init__.__contact__
-url = attr: SPARQLWrapper.__init__.__url__
+author = Ivan Herman, Sergio Fernández, Carlos Tejo Alonso, Alexey Zakhlestin
+author_email = rdflib-dev@googlegroups.com
+url = http://rdflib.github.io/sparqlwrapper
```

Comment:
Is it not maybe best to remove these values from `SPARQLWrapper/__init__.py`?

Reply:
adfa35f

Reply:
thanks